factorial_without_recrussion.js

```diff
@@ -0,0 +1,9 @@
+function factorial(x){
```

Comment:
please don't forget to use white spaces between an operators (for more prettifier view)

Reply:
Done